src/specialOffer/SpecialOffer.style.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components'
+
+import { space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+
+export const StyledSpecialOffer = styled.div``
+
+export const StyledSpecialOfferPicture = styled.img`
```

Comment:
you could use `figure` to wrap the image itself

Reply:
Done, thanks!